submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/service/SubmissionService.kt

```diff
@@ -35,34 +35,30 @@ class SubmissionService(
     private val eventsPublisherService: EventsPublisherService,
     private val rabbitTemplate: RabbitTemplate
 ) {
-    fun submit(request: SubmissionRequest): ExtSubmission {
-        return submissionSubmitter.submit(request)
-    }
+    fun submit(rqt: SubmitRequest): ExtSubmission = submissionSubmitter.submit(rqt)
 
-    fun submitAsync(request: SubmissionRequest) {
-        val accNo = request.accNo
-        logger.info { "$accNo ${request.owner} Received async submit request for submission $accNo" }
+    fun submitAsync(rqt: SubmitRequest) {
+        logger.info { "${rqt.accNo} ${rqt.owner} Received async submit request for submission ${rqt.accNo}" }
 
-        val (extSub, mode, draftKey) = submissionSubmitter.submitAsync(request)
+        val (accNo, version) = submissionSubmitter.submitAsync(rqt)
         rabbitTemplate.convertAndSend(
             BIOSTUDIES_EXCHANGE,
             SUBMISSIONS_REQUEST_ROUTING_KEY,
-            SubmissionRequestMessage(extSub.accNo, extSub.version, mode, extSub.owner, draftKey)
+            SubmissionRequestMessage(accNo, version)
         )
     }
 
     @RabbitListener(queues = [SUBMISSION_REQUEST_QUEUE], concurrency = "1-2")
     fun processSubmission(request: SubmissionRequestMessage) {
-        val (accNo, version, fileMode, submitter, draftKey) = request
-        logger.info { "$accNo $submitter Received process message for submission $accNo, version: $version" }
+        val (accNo, version) = request
+        logger.info { "Received process message for submission $accNo, version: $version" }
```

Comment:
Every log message should start with the accNo and submission owner whenever possible